modules/helm/helm.mk

```diff
@@ -83,35 +81,24 @@ helm_tool_footer_search ?= ^<!-- /AUTO-GENERATED -->
 ## @category [shared] Generate/ Verify
 generate-helm-docs: | $(NEEDS_HELM-TOOL)
 	$(HELM-TOOL) inject -i $(helm_chart_source_dir)/values.yaml -o $(helm_chart_source_dir)/README.md --header-search "$(helm_tool_header_search)" --footer-search "$(helm_tool_footer_search)"
-else
-.PHONY: generate-helm-docs
-## Generate Helm chart documentation.
-## @category [shared] Generate/ Verify
-generate-helm-docs: | $(NEEDS_HELM-DOCS)
-	$(HELM-DOCS) $(helm_chart_source_dir)/
```

Comment:
Could `NEEDS_HELM-DOCS` and/or `HELM-DOCS` be deleted, or is it still in use?

Reply:
Indeed, I removed them here: https://github.com/cert-manager/makefile-modules/pull/218/files#diff-cd4027dc82a354f5f35bd8a151160a66163403b3436ccac88e05c555ffbbca78L98-L99

Reply:
Ok, thanks. Is `NEEDS_xyz` a built-in make feature, or some convention used here?

Reply:
It is a variable that we create based on the values in the tools variable.